lib/power_action_utils.pm

```diff
@@ -81,12 +81,19 @@ sub reboot_x11 {
     my ($self) = @_;
     wait_still_screen;
     if (check_var('DESKTOP', 'gnome')) {
-        if (is_tumbleweed) {
+        # Get gnome version
+        x11_start_program('xterm -geometry 150x45+5+5', target_match => 'xterm');
```

Comment:
`x11_start_program('xterm')` should be enough. There should really be no need to make it this complicated. I have seen similar lines in test code which at best is exchanged as well. The line `x11_start_program('xterm')` is much more common

Reply:
+1